flight/Modules/VtolPathFollower/vtol_follower_fsm.c

```diff
@@ -80,7 +80,7 @@ enum vtol_fsm_event {
  * sets the nav mode and a default methods will farm it out to the appropriate
  * algorithm.
  */
-enum vtol_fsm_state {
+enum vtol_fsm_state_num {
```

Comment:
honestly?  Because I named the structure vtol_fsm_state below (because it made more sense to me) and then had to change the name of this.  Glad to put back if it's particularly objectionable.